extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationScrollToLink.js

```diff
@@ -0,0 +1,41 @@
+/*global require*/
+require(
+	['wikia.scrollToLink', 'wikia.window', 'jquery', 'wikia.browserDetect'],
+	function(scrollToLink, win, $, browserDetect) {
+		'use strict';
+		var offset = 0;
+
+		/**
+		 * @desc handler for hashchange event
+		 * @param {Event} event
+		 */
+		function hashChangeHandler(event) {
+			if (scrollToLink.handleScrollTo(win.location.hash, offset) || browserDetect.isIOS7orLower()) {
+				// prevent only if we managed to scroll to desired ID
+				event.preventDefault();
+				return false;
+			}
+		}
+
+		/**
+		 * @desc Function for third party extensions - bleed it to window
+		 * @param {Element} element
+		 */
+		win.GlobalNavigationScrollToElement = function(element) {
+			scrollToLink.scrollToElement(element, offset);
+		};
+
+
+		$(function() {
+			var $win = $(win),
+				spacingBelowGlobalNav = 10,
+				$globalNavigation = $('#globalNavigation'),
```

Comment:
This assignment is unnecessary since it's only used in the assignment below. We can just use `globalNavigationHeight = $('#globalNavigation').outerHeight(true);`